mkt/langpacks/views.py

```diff
@@ -60,3 +71,48 @@ def get_serializer_class(self):
             return LangPackUploadSerializer
         else:
             return LangPackSerializer
+
+
+@allow_cross_site_request
+def manifest(request, uuid):
+    """Returns the "mini" manifest for a langpack."""
+    try:
+        uuid_hex = UUID(uuid).hex
+    except ValueError:
+        raise Http404
+
+    langpack = get_object_or_404(LangPack, pk=uuid_hex)
+
+    if langpack.active or action_allowed(request, 'LangPacks', '%'):
```

Comment:
Yeah, typically we add a migration. And then we manually add users to the group.